### DISCUSSION TITLE
Will it be Optimised?

### DISCUSSION THREAD

**Discussion by yashminbhardwaj** [Spyxpo/web-to-app#31] [Q&A] [answered]
I use SWAB on a daily basis for my Freelance work, a great tool offered by Spyxpo.

I was wondering if it will be optimised to install, run and build faster with new binaries releasing on 1st Jan 2023?

It will be great if it can reduce installation time and build time.

**Comment by mantreshkhurana** [comment, marked as answer]
Yes it will be optimised. 

We are migrating multiple plugins and core components of SWAB to different and faster algorithms so that users can experience a better app builder.
